src/trpc/routers/_app.ts

```diff
@@ -82,6 +82,7 @@ export const appRouter = createTRPCRouter({
       })
     )
     .query(async ({ input }) => {
+      const limit = !input.cursor ? input.limit * 2 : input.limit;
```

Comment:
move it up by changing 6 to 12, or if you're open to it, create a global variable for the number of claims/bounties per page